.github/workflows/CI-pipeline.yml

```diff
@@ -334,9 +334,17 @@ jobs:
       - name: Start the E2E tests
         working-directory: ./manifests
         run: |
+          sudo apt update && sudo apt install -y wget tar
+          wget -q https://github.com/mikefarah/yq/releases/download/v4.27.2/yq_linux_amd64.tar.gz -O - |\
+          tar xz && mv yq_linux_amd64 yq
+
+          NAME_HASH="testing-framework-${SHORT_GITHUB_SHA}-${GITHUB_RUN_NUMBER}" ./yq e -i '(select(.kind == "ClusterRoleBinding").metadata.name = strenv(NAME_HASH))' ./testing-framework/testing-framework.yaml 
```

Comment:
Isn't the https://github.com/berops/claudie/blob/a7005d5e24e6cbaf6f98f8dd7bf628b6958a4786/.github/workflows/CI-pipeline.yml#L268 doing the same thing?

Reply:
yes, exactly.